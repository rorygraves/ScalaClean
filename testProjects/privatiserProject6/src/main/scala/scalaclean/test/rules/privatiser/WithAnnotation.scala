package x1.scalaclean.test.rules.privatiser.withannotation

object Entry extends App {
  (new Entry).foo
}

@deprecated
class Entry {

  @deprecated
  def foo: Int = foo + 1

}
