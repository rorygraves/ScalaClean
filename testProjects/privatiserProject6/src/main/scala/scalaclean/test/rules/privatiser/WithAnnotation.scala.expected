package x1.scalaclean.test.rules.privatiser.withannotation

object Entry extends App {
  (new Entry).foo
}

@deprecated
private[withannotation] class Entry {

  @deprecated
  private def foo: Int = foo + 1

}
